conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptionHandler.java

```diff
@@ -16,37 +16,14 @@
 
 package com.palantir.conjure.java.undertow.runtime;
 
-import com.palantir.conjure.java.api.errors.SerializableError;
-import com.palantir.conjure.java.undertow.lib.Serializer;
-import io.undertow.server.HttpHandler;
+import com.palantir.conjure.java.undertow.lib.ExceptionHandler;
 import io.undertow.server.HttpServerExchange;
 
-/**
- * Delegates to the given {@link HttpHandler}, and catches&forwards all {@link Throwable}s. Any exception thrown in the
- * delegate handler is caught and serialized using the Conjure JSON format into a {@link SerializableError}. The result
- * is written it into the exchange's output stream, and an appropriate HTTP status code is set.
- */
-final class ConjureExceptionHandler implements HttpHandler {
-
-    private final Serializer<SerializableError> serializer;
-    private final HttpHandler delegate;
-
-    ConjureExceptionHandler(HttpHandler delegate) {
-        this(delegate, ConjureExceptions.serializer());
-    }
-
-    // Constructor allows new exception handlers to be created without creating new serializer instances.
-    ConjureExceptionHandler(HttpHandler delegate, Serializer<SerializableError> serializer) {
-        this.delegate = delegate;
-        this.serializer = serializer;
-    }
+enum ConjureExceptionHandler implements ExceptionHandler {
```

Comment:
I think it will be a bit annoying for us to re-use this logic internally if we keep it package private

Reply:
Ya, I'm not too worried about opening visibility since it won't be useful/tempting to anyone else.